src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/ApplicationService.kt

```diff
@@ -117,6 +117,7 @@ class ApplicationService(
         reservedSlot = isReservedSlot,
         visitType = sessionTemplate.visitType,
         restriction = createApplicationDto.applicationRestriction.getVisitRestriction(),
+        userType = createApplicationDto.userType,
```

Comment:
should this go into App Insights as well?

Reply:
different ticket